angular-ngrx-scss/src/app/state/user/user.effects.spec.ts

```diff
@@ -64,4 +76,101 @@ describe('UserEffects', () => {
       done();
     });
   });
+
+  it('should get the user gists from the Github API', (done) => {
+    const userStateMock: UserState = {
```

Comment:
I'd recommend moving this to the top of the file and renaming it as "initialUserState" or something similar. That way you can reuse it in these checks easier. (Could also potentially import the initialUserState from the reducer or state file, whichever it lives in.

- [ ] Make this a file-wide variable you can reuse, or pull in the initialUserState variable to use instead